airbyte-integrations/connectors/destination-redshift/src/test-integration/java/io/airbyte/integrations/destination/redshift/RedshiftStagingS3DestinationAcceptanceTest.java

```diff
@@ -214,13 +215,18 @@ protected void setup(final TestDestinationEnv testEnv) throws Exception {
     getDatabase().query(ctx -> ctx.execute(createUser));
     final JsonNode configForSchema = Jsons.clone(baseConfig);
     ((ObjectNode) configForSchema).put("schema", schemaName);
+    TEST_SCHEMAS.add(schemaName);
```

Comment:
There are some test setups that create new schemas, so now `setup()` is passed TEST_SCHEMAS too.